packages/google-analytics/README.md

```diff
@@ -1,10 +1,67 @@
-> ## Beta version
-> Please, bear in mind that this package is currently in beta version. It hasn't proper tests or docs and its implementation may change with the final release.
+# `@frontity/google-analytics`
 
-# Google Analytics
-This package is used to add Google Analytics tracking codes to your Frontity project.
+[![Version](https://img.shields.io/npm/v/@frontity/google-analytics.svg)](https://www.npmjs.com/package/@frontity/google-analytics) [![npm](https://img.shields.io/npm/dw/@frontity/google-analytics)](https://www.npmjs.com/package/@frontity/google-analytics) [![License: Apache--2.0](https://img.shields.io/badge/license-Apache%202-lightgrey)](https://github.com/frontity/frontity/blob/master/LICENSE)
+
+Analytics package to use Google Analytics with Frontity
+
+
+## Install
+
+```sh
+npm i @frontity/google-analytics
+```
 
 ## Usage
-You can install it with npm:
 
-`npm i @frontity/google-analytics`
+Once is installed the GA Tracking ID should be added in the `frontity.settings.js` under the package `state`
+
+**`frontity.settings.js`**
+```js
+export default {
+  packages: [
+    {
+      name: "@frontity/google-analytics",
+      state: {
+        googleAnalytics: {
+          trackingIds: ['UA-12345678-9']
```

Comment:
For the basic usage I would show the singular option `trackingId`:
- https://github.com/frontity/frontity/blob/dev/packages/google-analytics/types.ts#L30

The plural `trackingIds` is only needed for advanced sites that need more than one tracking ID.

Reply:
Done! 👍